db/system_distributed_keyspace.cc

```diff
@@ -73,23 +73,6 @@ schema_ptr view_build_status() {
     return schema;
 }
 
-/* An internal table used by nodes to exchange CDC generation data. */
-schema_ptr cdc_generations() {
-    thread_local auto schema = [] {
-        auto id = generate_legacy_id(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION);
-        return schema_builder(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION, {id})
-                /* The timestamp of this CDC generation. */
-                .with_column("time", timestamp_type, column_kind::partition_key)
```

Comment:
What about mixed clusters?

Reply:
Removing this code causes the node to no longer attempt to announce the table creation when it starts. It won't cause the table, if it exists, to disappear (this is a normal distributed table, the only special thing about it is that the schema mutations of this table use `min_timestamp`). Thus in mixed clusters the table will still be there. In new clusters there will be no node to announce it so the table won't appear.